### PR TITLE
Our Teachers section added

### PR DESCRIPTION
**Pull request by andriignedash**
Note: some classes from "HC-7. Add About Us section" PR are used

### CODE REVIEW

**Comment by andriignedash** [comment]
[Our teachers Section](https://trello.com/c/R4Syfhz5/14-our-teachers-section)

File: src/css/common/card-set.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@media only screen and (min-width: 768px) {
+  .card-set {
+    row-gap: 32px;
+  }
+}
+
+.card-set-item {
+  width: calc((100% - 32px * (var(--items) - 1)) / var(--items));
+  display: flex;
+    flex-direction: column;
```
Comment (VOSolyanik) [inline src/css/common/card-set.css]: spacing

File: src/css/sections/teachers.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+@media only screen and (min-width: 1152px) {
+  .teachers-list-item.card-set-item {
+    --items: 4;
+  }
+}
+
+.teachers-list-item-photo {
+  border-radius: 500px;
```
Comment (VOSolyanik) [inline src/css/sections/teachers.css]: border-radius: 50%;

File: src/css/sections/teachers.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  width: 150px;
+  height: 150px;
+  margin-bottom: 16px;
+}
+
+.teachers-list-item-name {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 120%;
```
Comment (VOSolyanik) [inline src/css/sections/teachers.css]: 1.2

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

**Review by VOSolyanik — APPROVED** [review]

**Pull request merged**